addons/medical_ai/functions/fnc_healingLogic.sqf

```diff
@@ -324,7 +346,7 @@ if (true) then {
         };
 
         // If all limbs are tourniqueted, bandage the one with the least amount of wounds, so that the tourniquet can be removed
-        _bodyPart = call _fnc_findNoTourniquet;
+        _bodyPart = false call _fnc_findNoTourniquet;
```

Comment:
we'll hit line 333 first, so I guess it could be simplified to this
but doesn't really hurt to keep it symmetrical with epi logic

```suggestion
        _bodyPart = selectRandom ["leftarm", "rightarm", "leftleg", "rightleg"]
```

Reply:
What should happen if a unit doesn't have enough bandages to stop bleeding on a tourniqueted limb: Should they be able to give themselves morphine or not? Atm it's unintentionally not possible for AI to give themselves morphine if they have tourniquets on.
I will be changing the logic so that they can, if they are out of bandages. This would mean that the suggestion above isn't necessary.